src/abstract_register.jl

```diff
@@ -73,6 +73,19 @@ If only an integer is provided, then returns a lambda function.
 
 addbits!(n::Int) = @λ(register -> addbits!(register, n))
 
+"""
+    insert_qubits!(register, loc::Int; n::Int=1) -> register
+    insert_qubits!(loc::Int; n::Int=1) -> λ(register)
+
+Insert `n` qubits to given register in state |0>.
+i.e. |psi> -> |psi> ⊗ |000> ⊗ |psi>, increased bits have higher indices.
+
+If only an integer is provided, then returns a lambda function.
+"""
+@interface insert_qubits!(::AbstractRegister, loc::Int; n::Int=1)
+insert_qubits!(loc::Int, n::Int=1) = @λ(register -> insert_qubits!(register, loc; n=n))
```

Comment:
This is not consistent, `n` is a keyword for other interfaces.

Reply:
nice catch